conan/internal/api/install/generators.py

```diff
@@ -32,7 +32,8 @@
                "SConsDeps": "conan.tools.scons",
                "QbsDeps": "conan.tools.qbs",
                "QbsProfile": "conan.tools.qbs",
-               "CPSDeps": "conan.tools.cps"
+               "CPSDeps": "conan.tools.cps",
+               "Ament": "conan.tools.ros"
```

Comment:
If this generator generates information about dependencies, maybe ``AmentDeps`` is more aligned with the other generators?